package.json

```diff
@@ -8,26 +8,30 @@
     "test": "jest --ci --forceExit"
   },
   "devDependencies": {
+    "babel-core": "6.26.0",
+    "babel-jest": "22.4.3",
+    "babel-preset-env": "1.6.1",
```

Comment:
Even though there's no ES6 or async usage in the test code here, there's a lot of it in the Node SDK shared test code that we're calling, so we need Babel to be able to build with Node 6.